it's a sequelize custom validation and that throw new Error should be handled by sequelize